tests/zfs-tests/include/libtest.shlib

```diff
@@ -31,12 +31,14 @@
 
 . ${STF_TOOLS}/include/logapi.shlib
 
-ZDB=${ZDB:-/sbin/zdb}
-ZFS=${ZFS:-/sbin/zfs}
-ZINJECT=${ZINJECT:-/sbin/zinject}
-ZHACK=${ZHACK:-/sbin/zhack}
-ZPOOL=${ZPOOL:-/sbin/zpool}
-ZTEST=${ZTEST:-/sbin/ztest}
+exec_prefix=${exec_prefix:-/}
+sbindir=$exec_prefix/sbin
+ZDB=${ZDB:-$sbindir/zdb}
+ZFS=${ZFS:-$sbindir/zfs}
+ZINJECT=${ZINJECT:-$sbindir/zinject}
+ZHACK=${ZHACK:-$sbindir/zhack}
+ZPOOL=${ZPOOL:-$sbindir/zpool}
+ZTEST=${ZTEST:-$sbindir/ztest}
```

Comment:
Perhaps we could drop this block entirely if the sourced files reordered in `default.cfg` which also defines these.